upup/pkg/fi/cloudup/scaleway/cloud.go

```diff
@@ -224,7 +227,7 @@ func (s *scwCloudImplementation) FindVPCInfo(id string) (*fi.VPCInfo, error) {
 
 func (s *scwCloudImplementation) GetApiIngressStatus(cluster *kops.Cluster) ([]fi.ApiIngressStatus, error) {
 	klog.V(8).Info("Scaleway doesn't have load-balancers yet so GetApiIngressStatus is not implemented")
```

Comment:
Does this mean that Scaleway doesn't have LBs at all, or that this was not yet implemented for kOps?

Reply:
Load-balancers will be implemented in an other PR but they are ready. I reworded this message too.

Reply:
Perfect.